### PR TITLE
Ed branch 1

### PR DESCRIPTION
**Pull request by edrob999**
Updated community Readme - simplifying the document, putting info about the umbrella project at top of file
signed-off-by Ed Robinson <ed.robinson@gmail.com>

### CODE REVIEW

**Review by orville-wright — APPROVED** [review]
LGTM
Approved

**Comment by orville-wright** [comment]
Removed @tiagolobocastro and @avishnu reviews. Not critical ATM.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
-# Welcome to our community...
-[![OpenEBS Welcome Banner](/images/openebs_community_banner_retro_gamer.png)](https://www.openebs.io/)
-
-## [openebs.io](https://www.openebs.io/)
+# OpenEBS Community<BR>
+> [!Important]
+> OpenEBS is an umbrella project, composed as a federation of individual sub projects (repositories). The project files in this repository apply to the umbrella project, every sub project, repository and file in the OpenEBS organization.
```
Comment (avishnu) [inline README.md:3]: "sub projects (repositories") needs to change to "sub projects and concerned/related repositories"

File: README.md [hunk 2]
```diff
@@ -17,22 +21,19 @@ Currently is is being curated by...
...
-<BR>
-<BR>
-
-> [!IMPORTANT]
-> We are a community of innovative Data Storage engineers that contribute to this project. We are greatful for the Major sponsorship that is provided by [DataCore](https://datacore.com), who donates a large team of dedicated Dev/QA/Engineers, Product Mgmt, Marketing, Business and Operational leadership as well as significant engineering infrastructure rescources and financial funding. Without DataCore, Inc... OpenEBS would not be the successful product that is it today. OpenEBS is teh result of 100's of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. - All who are passionate about storage and Data are welcome here.
->
-
-<BR>
-<BR>
+## Who we are
```
Comment (avishnu) [inline README.md:24]: Typos
1. that it is today.
2. hundreds instead of 100's

File: README.md [hunk 3]
```diff
@@ -17,22 +21,19 @@ Currently is is being curated by...
...
+[![OpenEBS Welcome Banner](/images/openebs_community_banner_retro_gamer.png)](https://www.openebs.io/)
 
 > [!NOTE]
 > CNCF Project status : ARCHIVE <BR>
-> In Jan 2024, under the guidnace of CNCF TOC, our project moved from Sandbox level to Archvie level so that we could impliemnt an extensive restructing initative. We are actively archiving out old projects, repos and dependancies to a CNCF owned repo called: OpeneEBS-Archive (https://github.com/openebs-archive).<BR>
-> The product team is unifying 4 main repos (Mayastor, LocalPV-LVM, LocalPV-ZFS, LocalPV-Hostpath) and their support technbology stack repos into one core OpnenEBS code-base. See our main repo for more details.
+> In Jan 2024, under the guidance of CNCF TOC, our project moved from Sandbox level to Archive level, enabling us to implement an extensive restructing initative. We are actively working to archive old projects, repos and dependancies to a CNCF owned repo called: OpeneEBS-Archive (https://github.com/openebs-archive), plus update our governance and operating procedures.<BR>
```
Comment (avishnu) [inline README.md:30]: archive legacy sub projects, repositories and dependencies to a CNCF-owned GitHub organization called ...

File: README.md [hunk 4]
```diff
@@ -17,22 +21,19 @@ Currently is is being curated by...
...
+> What's changing in the technology stack? The product team is unifying 4 main repos (Mayastor, LocalPV-LVM, LocalPV-ZFS, LocalPV-Hostpath) and their technology stack repos into one core OpnenEBS code-base.
```
Comment (avishnu) [inline README.md:31]: unifying 4 major sub projects (..)...

File: README.md [hunk 5]
```diff
@@ -17,22 +21,19 @@ Currently is is being curated by...
...
+> For more information see See [the parent issue]( https://github.com/openebs/openebs/issues/3701) in our main repo for more detail.
```
Comment (avishnu) [inline README.md:32]: "see" is repeated

**Pull request merged**